### PR TITLE
update amr_grid.inp

### PR DESCRIPTION
**Pull request by willsonho2000**
nx, ny, nz should be integers, otherwise it will show up "Bad integer" when running the code

### CODE REVIEW

File: yt_astro_analysis/radmc3d_export/RadMC3DInterface.py [hunk 1]
```diff
@@ -232,10 +232,10 @@ def write_amr_grid(self):
             ix = int(ind[0]+0.5)
             iy = int(ind[1]+0.5)
             iz = int(ind[2]+0.5)
+            nx, ny, nz = layer.ActiveDimensions / 2
```
Comment (matthewturk) [inline yt_astro_analysis/radmc3d_export/RadMC3DInterface.py:235]: Would it be possible to do `layer.ActiveDimensions // 2` ?  I think that might eliminate the need to cast to `int` at all.

**Review by matthewturk — APPROVED** [review]
Thank you for submitting this!  I've left an optional comment, but otherwise this looks good to me.

File: yt_astro_analysis/radmc3d_export/RadMC3DInterface.py (hunk 1 above)
Comment (willsonho2000) [reply yt_astro_analysis/radmc3d_export/RadMC3DInterface.py:235]: Yes, that would be better. Thanks a lot!

Comment (willsonho2000) [reply yt_astro_analysis/radmc3d_export/RadMC3DInterface.py:235]: Sorry, I noticed that the type of nx, ny, nz won't change if only change the code to "layer.ActiveDimensions // 2".
It remains "float", so maybe it need another method.

**Review by brittonsmith — APPROVED** [review]
Thanks for this. I'll merge it.

File: yt_astro_analysis/radmc3d_export/RadMC3DInterface.py (hunk 1 above)
Comment (brittonsmith) [reply yt_astro_analysis/radmc3d_export/RadMC3DInterface.py:235]: Yes, evidently `//` will return a float if either the numerator or denominator is a float. I was surprised by this, too.

**Pull request merged**